CHANGELOG.md

```diff
@@ -4,8 +4,13 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Added a configuration to specify the port numbers for `http`, `https` and `metrics` ([#XX]).
+
 ### Changed
 
+- BREAKING: Ensure no unknown config properties have been set. This is to make the user aware that what he tried to configure is not a valid configuration. You may need to adopt your configuration and remove any unknown properties ([#XX]).
```

Comment:
```suggestion
- BREAKING: Ensure no unknown config properties have been set. This is to make the user aware that what he tried to configure is not a valid configuration. You may need to adapt your configuration and remove any unknown properties ([#43]).
```

Reply:
You'll need to add `[#43]: https://github.com/stackabletech/trino-lb/pull/43`, as the line is beyond where I can make a suggestion in Github Issues.

Reply:
The suggestion wasn't applied

Reply:
Sorry, I only saw the PR number. typo is fixed